README.md

```diff
@@ -13,7 +13,7 @@ If you have Homebrew (https://brew.sh) you can install Maven via:
 
 To build run:
 
-`mvn clean package assembly:single javadoc:javadoc pmd:pmd pmd:cpd checkstyle:check`
```

Comment:
Marc, check whether we need to manually run pmd and checkstyle after my changes to the pom.
Looking at the pom, checkstyle is bound to the Validate stage in the maven build cycle, so I think it will run before compile. Just need to confirm that it is running before removing that instruction from the build command.
I don't think the pmd or javadoc targets are mapped to an execution, which is fine.

Reply:
It looks like checkstyle is automatically firing at the validate stage, so we can remove the checkstyle:check from this command:
randyw$ mvn clean compile | grep -i checkstyle
[INFO] --- maven-checkstyle-plugin:3.0.0:check (validate) @ lds-tools-export ---
us-aus-wireless-10-0-144-218:wlts-import